client/src/main/java/com/influxdb/client/OrganizationsQuery.java

```diff
@@ -0,0 +1,96 @@
+package com.influxdb.client;
+
+import javax.annotation.Nullable;
+
+public class OrganizationsQuery {
+
+    /**
+     * offset  (optional)
+     */
+    @Nullable
+    private Integer offset;
+
+    /**
+     * limit  (optional, default to 20)
+     */
+    @Nullable
+    private Integer limit;
+
+    /**
+     * descending  (optional, default to false)
+     */
+    @Nullable
+    private Boolean descending;
+
+    /**
+     * Filter organizations to a specific organization name. (optional)
+     */
+    @Nullable
+    private String org;
+    /**
+     * Filter organizations to a specific organization ID. (optional)
+     */
+    @Nullable
+    private String orgID;
+
+    /**
+     * Filter organizations to a specific user ID. (optional)
+     */
+    @Nullable
+    private String userID;
+
+    @Nullable
+    public Integer getOffset() {
+        return offset;
+    }
+
+    public void setOffset(@Nullable Integer offset) {
+        this.offset = offset;
+    }
+
+    @Nullable
+    public Integer getLimit() {
+        return limit;
+    }
+
+    public void setLimit(@Nullable Integer limit) {
```

Comment:
Add `final` modifier:

```suggestion
    public void setLimit(@Nullable final Integer limit) {
```